### PR TITLE
Adding to JMS Input and New JMS Output

### PR DESCRIPTION
**Pull request by AlphaCluster**
I added onto dclaveau's JMS Input and also created a new output using jruby-jms. The input now uses codecs as well for the JMS Message body if it is text.

### CODE REVIEW

File: logstash-contrib.gemspec [hunk 1]
```diff
@@ -39,6 +39,7 @@ Gem::Specification.new do |gem|
     gem.add_runtime_dependency "geoscript", "0.1.0.pre"           #(MIT license)
     gem.add_runtime_dependency "jruby-win32ole"                   #(unknown license)
     gem.add_runtime_dependency "jdbc-mysql"                       # For input drupal_dblog (BSD license)
+    gem.add_rutnime_dependency "jruby-jms"                        #(Apache 2.0 license)
```
Comment (skade) [inline logstash-contrib.gemspec]: Not sure if this PR is maintained, but here's a typo.

**Comment by jsvd** [comment]
I'd like to transform this PR into 2 plugins, now that with 1.5 we have separate repos per plugin: https://github.com/logstash-plugins/

I created the 2 repos under logstash-plugins:
input https://github.com/logstash-plugins/logstash-input-jms
output https://github.com/logstash-plugins/logstash-output-jms

Since the whole PR is 24 commits long and touches many many files (that are not relevant), can you do a direct PR for each repository (input and output), @AlphaCluster ?

**Comment by AlphaCluster** [comment]
Sure I can look into it this weekend.

**Comment by AlphaCluster** [comment]
https://github.com/logstash-plugins/logstash-input-jms/pull/1 Pull request for input.

**Comment by AlphaCluster** [comment]
https://github.com/logstash-plugins/logstash-output-jms/pull/1 Pull request for output.

**Comment by wiibaa** [comment]
@suyograo Dup of 
https://github.com/logstash-plugins/logstash-input-jms/pull/1 (merged)
https://github.com/logstash-plugins/logstash-output-jms/pull/1 **(open)**

**Pull request closed without merging**